src/lib.rs

```diff
@@ -85,7 +85,7 @@ impl BasicBlock {
     }
 }
 
-/// A MIR statement.
+/// A Statement.
```

Comment:
Should be lower-case "statement"? [As a minor issue, I'm also not sure that this comment is worth the line it takes up!]

Reply:
I tend to agree that this comment can go.

[This is a habit I've picked up from rustc, where it sometimes insists that every item is documented]